GoogleChromeLabs/ps-analysis-tool
# Refactor: Increase test coverage.

## Description
This PR aims to introduce tests to increase testing coverage in the develop branch. Various test cases have been introduced to add test coverage to utilities and components. 

## Testing Instructions

Note: This PR doesn't introduce any UI changes so it doesn't need any QA.
- Open the terminal and run `npm run test`.
- Now run `npm run test:coverage` you can see a generated test coverage which covers 85% of the project.
